### PR TITLE
frontend: add guide entry about change output

### PR DESCRIPTION
**Pull request by thisconnect**
Added guide entry to explain when the change output is a Taproot,
only shown in the guide of bitcoin accounts.

The rest was updated in locize and will need a locize-pull

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -763,6 +763,10 @@
       }
     },
     "send": {
+      "change": {
+        "text": "The change will be returned to a Native Segwit address, unless you have at least one Taproot UTXO. In that case the change will be returned to a Taproot address.\n\nPlease note, if you are using coin control and select only Native Segwit UTXOs, the change will be returned to a Native Segwit address, regardless of whether you have a Taproot UTXO in that account. However, if you select Wrapped Swegit UTXOs, the change will still be returned to a Native Segwit address.",
```
Comment (benma) [inline frontends/web/src/locales/en/app.json:767]: typo: Wrapped Swegit -> Wrapped Segwit

@jadzeidan this feels very convoluted, is there no easier and shorter way? How about:

"The change will be returned to a Taproot address if you have at least one Taproot UTXO.  If you use coin control, the change will be returned to a Taproot address if there is at least one Taproot UTXO among the selected UTXOS. In all other cases, the change is returned to a Native Segwit address."

**Comment by benma** [comment]
> The rest was updated in locize and will need a locize-pull

image


Why drop `coinName`? In Litecoin the xpubs and this guide entry are also shown.

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (jadzeidan) [reply frontends/web/src/locales/en/app.json:767]: Sure, fine with me.

Comment (thisconnect) [reply frontends/web/src/locales/en/app.json]: fixed and pushed

**Comment by thisconnect** [comment]
> Why drop coinName? In Litecoin the xpubs and this guide entry are also shown.

Oversight by me, but reading the new sentence that has Taproot:

"Each xpub is tied to the "Type" shown: either "Native Segwit (bech32)" , "Wrapped Segwit" or “Taproot”. These are script types used by Bitcoin" 

Writing "used by Litecoin" would be wrong as it doesn't support Taproot.

**Comment by benma** [comment]
> > Why drop coinName? In Litecoin the xpubs and this guide entry are also shown.
> 
> Oversight by me, but reading the new sentence that has Taproot:
> 
> "Each xpub is tied to the "Type" shown: either "Native Segwit (bech32)" , "Wrapped Segwit" or “Taproot”. These are script types used by Bitcoin"
> 
> Writing "used by Litecoin" would be wrong as it doesn't support Taproot.

in BB01 there is also P2PKH ;)

What about "Each xpub is tied to the "Type" shown, for example "Native Segwit (bech32)" or "Wrapped Segwit".

**Comment by thisconnect** [comment]
cc @jadzeidan @Stadicus PTAL

**Comment by Stadicus** [comment]
I can't judge the technical aspects, but the text looks good to me. :)

**Review by benma — APPROVED** [review]

**Comment by jadzeidan** [comment]
> What about "Each xpub is tied to the "Type" shown, for example "Native Segwit (bech32)" or "Wrapped Segwit".

Sounds good to me.

**Pull request merged**